stacks/carpark-stack.js

```diff
@@ -0,0 +1,125 @@
+import {
+  Bucket,
+  Function,
+  Queue,
+  EventBus
+} from '@serverless-stack/resources'
+import { Duration } from 'aws-cdk-lib'
+import * as sqs from 'aws-cdk-lib/aws-sqs'
+
+import { CARPARK_EVENT_BRIDGE_SOURCE_EVENT } from '../api/carpark/event-bridge/index.js'
+
+import { getConfig } from './config.js'
+
+/**
+ * @param {import('@serverless-stack/resources').StackContext} properties
+ */
+export function CarparkStack({ stack }) {
+  // @ts-expect-error "prod" | "dev" | "staging" only allowed for stage
+  const stackConfig = getConfig(stack.stage)
+
+  const carparkBucket = new Bucket(stack, 'car-store', {
+    ...stackConfig.bucketConfig,
+  })
+
+  // Elastic IPFS event for indexing
+  const indexerTopicQueue = new Queue(stack, 'indexer-topic-queue', {
+    cdk: {
+      queue: sqs.Queue.fromQueueArn(
+        stack,
+        'indexer-topic',
+        'arn:aws:sqs:us-west-2:505595374361:indexer-topic'
+      ),
+    },
+  })
```

Comment:
we should check in with @francardoso93 here... for pickup the request was to have it create an SNS topic that E-IPFS can subscribe to, rather than publishing directly to the existing queue. The preference is to have a topic per system, and have E-IPFS subscribe to them all.

Reply:
we can land it like this for now, but we might need to change it later.

Reply:
That would be cool! Let's not block this PR and create an issue to track it.

@francardoso93 is E-IPFS already ready to support this?